apps/dashboard/src/app/(internal)/event/[id]/payment-page.tsx

```diff
@@ -56,28 +58,51 @@ export const PaymentPage: FC = () => {
         cell: (info) => {
           const value = info.getValue()
 
-          if (value.paymentChargedAt) {
-            return <Badge color="green">Betalt</Badge>
-          }
-          if (value.paymentReservedAt) {
-            return <Badge color="blue">Betaling reservert</Badge>
-          }
+          let badge: BadgeProps = {}
+
           if (value.paymentRefundedAt) {
-            return <Badge color="indigo">Betaling refundert</Badge>
-          }
-          if (value.paymentId) {
-            return <Badge color="yellow">Venter på betaling</Badge>
+            badge = { color: "gray", children: "Refundert" }
+          } else if (value.paymentChargedAt) {
+            badge = { color: "green", children: "Betalt" }
+          } else if (value.paymentReservedAt) {
+            badge = { color: "blue", children: "Reservert" }
+          } else if (!value.paymentRefundedAt && value.paymentRefundedById) {
+            badge = { color: "gray", children: "Kansellert" }
+          } else if (value.paymentDeadline) {
+            badge = { color: "red", children: "Ikke betalt" }
+          } else {
+            badge = { color: "gray", children: "Ingen betaling" }
           }
```

Comment:
Ideally this should be sent from `rpc`, like we do with attendee status. But this is fine for now probably.